lms/djangoapps/grades/new/course_grade.py

```diff
@@ -240,6 +312,28 @@ def _log_event(self, log_func, log_statement):
             self.student.id
         ))
 
+    def _populate_course_structure_grades(self):
```

Comment:
We should remove this method in favor of just calling the existing `compute_and_update` method - which does the bulk operations that we want.